src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -397,7 +397,13 @@ private function createBlock($messages, $type = null, $style = null, $prefix = '
                 $message = OutputFormatter::escape($message);
             }
 
-            $lines = array_merge($lines, explode(PHP_EOL, wordwrap($message, $this->lineLength - $prefixLength - $indentLength, PHP_EOL, true)));
+            $length = $this->lineLength;
+
+            if (false !== strpos($message, '</comment>')) {
```

Comment:
Doesn't the issue also apply to other style tags, like `info` or  `question`? There's also `</>` and custom defined styles: https://symfony.com/doc/current/console/coloring.html

Reply:
I think we will go back to a regexp checking only the </> et counting that value. So it works with all tags.

Reply:
Thanks for the review btw @Destroy666x !